webpack/babel-loader-rule.js

```diff
@@ -15,7 +16,7 @@ const folioScopeBlacklist = [
 function babelLoaderTest(fileName) {
   const nodeModIdx = fileName.lastIndexOf('node_modules');
   if (fileName.endsWith('.js')
-    && (nodeModIdx === -1 || fileName.lastIndexOf('@folio') > nodeModIdx)
+    && (nodeModIdx === -1 || ['@folio', ...extraTranspile].reduce((acc, cur) => (fileName.lastIndexOf(cur) > nodeModIdx) || acc, false))
```

Comment:
That's a nicely compact `reduce` call, but it's rather opaque, as are the comments on lines 9-15. Could you maybe replace the comments with a few examples of values for which this function will return true and false, including an example of how `STRIPES_TRANSPILE_TOKENS` figures in?

Reply:
Certainly! How's that?